code/ui/generalhud/inspectmenu/InspectMenu.scss

```diff
@@ -1,119 +1,78 @@
 InspectMenu {
-    border: 1px solid #1b1b1b;
-    background-color: #000c;
+    background-color: #212121;
     color: #fafafa;
-    font-family: Poppins;
+    border-radius: 5px;
     font-weight: 500;
-    border-radius: 4px;
-    margin: auto;
-    top: 66%;
+    top: 60%;
+    left: 39.5%;
```

Comment:
Remind me to put up an issue to center this correctly, we need to wait for the `translate` `overflow` fix.